velox/exec/StreamingAggregation.cpp

```diff
@@ -171,6 +203,11 @@ RowVectorPtr StreamingAggregation::createOutput(size_t numGroups) {
     }
   }
 
+  if (sortedAggregations_) {
```

Comment:
Shouldn't we skip sorted aggs in the loop above?

Reply:
Fixed